### PR TITLE
FooBar (include examples)

### PR DESCRIPTION
**Pull request by AlexWeinstein92**
Related to https://github.com/reactific/riddl/pull/220

### CODE REVIEW

File: src/riddl/FooBarSameDomain/FooBar.conf [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  input-file = "examples/src/riddl/FooBarSameDomain/FooBar.riddl"
+  output-dir = "examples/target/translator/FooBarSameDomain"
+  project-name = "FooBar"
+  erase-output = true
+  base-url = "https://riddl.tech"
+  source-url = "https://github.com/reactific/riddl"
+  edit-path = "blob/main/examples/src/riddl/FooBarSameDomain"
+  site-logo-path = "images/FooBar.png"
+}
+
```
Comment (reid-spencer) [inline src/riddl/FooBarSameDomain/FooBar.conf:25]: could you please also add the below snippet at the end so we can `from ... validate` too? Thanks
```
validate {
   input-file = "examples/src/riddl/FooBarSameDomain/FooBar.riddl"
}```

**Review by reid-spencer — CHANGES_REQUESTED** [review]
LGTM, just one simple addition and I'll approve this.

File: src/riddl/FooBarSameDomain/FooBar.conf (hunk 1 above)
Comment (AlexWeinstein92) [reply src/riddl/FooBarSameDomain/FooBar.conf:25]: Done

**Review by reid-spencer — APPROVED** [review]
LGTM

**Pull request merged**